### ISSUE TITLE
Compare Screen Unclear

### ISSUE THREAD

**Issue by zcmoore** [paulsoiya/teamj#32]
The compare screen does not make it clear that it is displaying the player who's stats are closest to the user's.

We should add a label or text area that explains the purpose (e.g., "Congratulations! Your stats are close to [Mark Sanchez]!")

**Comment by zcmoore** [comment]
This should be refactored before caking on this change (it will need to be changed across multiple screens and controllers, with our current setup).

Based on the timeline, I'm thinking this will be left unfinished.

**Comment by tdscott10** [comment]
I don't think this a huge deal.  I think the name of our app should be apparent enough for them to figure out what is going on.  We can talk about this more and possibly just add a label on each fxml during the next sprint if we need to
